netsim/modules/_routing.py

```diff
@@ -339,7 +362,7 @@ def remove_unused_igp(node: Box, proto: str, remove_module: bool = True) -> bool
     return False                                                            # ... OK, we're good
 
   for vdata in node.get('vrfs',{}).values():                                # Is protocol active in at least one VRF?
-    if proto in vdata:
+    if proto in vdata and vdata[proto].interfaces:                          # ... and at least on one interface?
```

Comment:
```vdata[proto].get('interfaces')``` to avoid creating ```{}```?

Reply:
I don't think we'd get a protocol data structure in a VRF with no relevant interfaces, but OK